pkgs/development/compilers/mono/generic-cmake.nix

```diff
@@ -76,6 +76,10 @@ stdenv.mkDerivation rec {
   # [1] https://github.com/mono/mono/blob/master/scripts/gmcs.in
   + ''
     ln -s $out/bin/mcs $out/bin/gmcs
+  ''
+  # Make MSBuild stop complaining
+  + ''
+    cp $out/lib/mono/xbuild/14.0/Microsoft.Common.props $out/lib/mono/xbuild/15.0/Microsoft.Common.props
```

Comment:
Was this an error or a warning? If it was just a warning i think it’s best to not mess with this stuff.

Reply:
It was an error.

Reply:
Update on this: I cannot recall, but I *believe* (knowing me probably not) the upstream Mono maintainers told me to apply this. I completely forgot about this until yesterday, though, you might have to check.